client/src/plugins/zeebe-plugin/start-instance-plugin/__tests__/StartInstancePluginSpec.js

```diff
@@ -409,7 +409,7 @@ describe('<StartInstancePlugin> (Zeebe)', () => {
       const notificationHTML = shallow(notification.content).html().replace(/&amp;/g, '&');
 
       expect(notificationHTML).to.include(
-        'https://REGION.operate.camunda.io/CLUSTER_ID/instances/undefined'
+        'https://region.operate.camunda.io/CLUSTER_ID/instances/test'
```

Comment:
Guess this is automatically lower-cased due to using `URL` now?

Reply:
I'd propose to also change the input (region) in the process.

Reply:
Yup, that's how URL works. I will adjust the test data then.